Add case management images to docs (#21636)
DataDog/documentation: content/en/service_management/case_management/settings.md

Project handles can be used in monitors to automatically create cases. In the monitor message body, include `@case-<project_handle>`. Datadog suggests a handle based on the project's name. You can accept or modify it as you wish. 

> Navigate to [Project Settings][1] > Integrations > Datadog Monitors and then toggle it on to get your @case-<project_handle>.

Reviewer: ```suggestion
Navigate to the [Project Settings page][1], click **Integrations** > **Datadog Monitors**, and click on the toggle to get your @case-<project_handle>.
```
